demo/common/nodeeditor/node_type_output.c

```diff
@@ -0,0 +1,31 @@
+struct node_type_output {
+    struct node node;
+    struct nk_colorf input_val;
```

Comment:
`node_type_output` is more of a color output. Should we consider renaming it to `node_type_color` instead?